test/site/_collections/posts-with-lighthouse/pages/collection.md

```diff
@@ -0,0 +1,3 @@
+{%- for post in collections.postsWithLighthouse -%}
```

Comment:
What do you think about renaming this directory to `fixtures` so folks can tell at a glance what these are

Reply:
done